work/1 Table.sql

```diff
@@ -0,0 +1,17 @@
+use costumeDB
+go
+drop table if exists Costumes 
+go
+create table dbo.Costumes(
+CostumeId int not null identity primary key,
+CustomerName varchar (50) not null constraint ck_costumes_customername_cannot_be_blank check(customername <> ''),
+CostumesBought varchar (20) not null constraint ck_costumes_CostumesBought_cannot_be_blank check (CostumesBought <> ''),
```

Comment:
-- LL CostumesBought isn't such a clear column name.. change to costumename or costumedesc